packages/terra-notification-dialog/src/terra-dev-site/doc/example/ContentNotificationDialog.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Button from 'terra-button';
+import NotificationDialog, { NotificationDialogVariants } from '../../../NotificationDialog';
+
+const clickConfirm = () => {
+  console.log('You clicked confirm'); // eslint-disable-line no-console
+};
+
+class ContentNotificationDialog extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      isOpen: false,
+    };
+
+    this.handleOpenModal = this.handleOpenModal.bind(this);
+    this.handleCloseModal = this.handleCloseModal.bind(this);
+  }
+
+  handleOpenModal() {
+    this.setState({ isOpen: true });
+  }
+
+  handleCloseModal() {
+    this.setState({ isOpen: false });
+  }
+
+  render() {
+    return (
+      <div>
+        <NotificationDialog
+          variant={NotificationDialogVariants.ALERT}
+          isOpen={this.state.isOpen}
+          title="Make sure that the title relates directly to the choices."
+          startMessage="The Main Instruction is text used to provide more detail or define terminology. Don’t repeat the title verbatim."
```

Comment:
Can we make the start and end messages different?

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/899/commits/f95efef89b73c14e91d1b18e9543cd665f7a6191#diff-110d398cf6d3fcb0a08cbb16ae55fd55R43